### ISSUE TITLE
SyntaxError: Unexpected token .

### ISSUE THREAD

**Issue by div-wynk** [ng-bootstrap/ng-bootstrap#3774]
Please remember, the issues forum is __NOT__ for support requests. It is for bugs and feature requests only.
Please read https://github.com/ng-bootstrap/ng-bootstrap/blob/master/CONTRIBUTING.md and search
existing issues (both open and closed) prior to opening any new issue and ensure you follow the instructions therein.

### Bug description:
While trying to run SSR on my angular web app I am getting this issue..

module.exports = ../../../../node_modules/@ng-bootstrap/ng-bootstrap;
                 ^

SyntaxError: Unexpected token .
    at Module._compile (internal/modules/cjs/loader.js:721:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)

### Link to minimally-working StackBlitz that reproduces the issue:
its a prod app!

You can fork a StackBlitz from one of our [demos](https://ng-bootstrap.github.io/#/components) and use it as a starting point.

Please note that we can _not_ act on bug reports without a _minimal_ reproduction scenario in a StackBlitz. Here is why:
https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug

### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 9

ng-bootstrap: 6.1.0

Bootstrap: 4.4.1

**Comment by benouat** [comment]
I am not that familiar with the `SSR` aspect, but I know we do have an application dedicated to that, for us to test during CI that it works, for all widgets. cc @maxokorokov Am I right ?

Outcome is that we don't have these issues. 
Did you had a look at it ?

**Comment by benouat** [comment]
Keep in mind, also, that you can't use issues as support request. You should use Stack Overflow for that. Thanks

**Comment by maxokorokov** [comment]
@div-wynk I guess we're going to need more info on this one, please. We're going to need some kind of reproduction scenario anyway.

If you need to look at how exactly we're testing SSR, you can take a look at the app we're using:
- https://github.com/ng-bootstrap/ng-bootstrap/tree/master/ssr-app, it is a part of this repo
- more info on useful commands can be found ([in this doc](https://github.com/ng-bootstrap/ng-bootstrap/blob/master/DEVELOPER.md#project-structure))

Basically you have to:
- clone this repo
- run `yarn` to install deps
- run `ng build ng-bootstrap --prod` to build the library
- run `yarn ssr` to run our ssr app with tests

I'm not sure if it will help you, but at least you can check what we're doing and all the configs.

**Comment by benouat** [comment]
Closing as no activity/feedback
